### ISSUE TITLE
ISBAT identify an external link within the Map block Google API Settings PanelBody

### ISSUE THREAD

**Issue by richtabor** [godaddy/coblocks#535]
Users should be able to recognize an external link within the UI.
The "Retrieve your key" and "Need help?" links within the Google Maps API Key PanelBody should be rendered with the core `ExternalLink` component ([example](https://gist.github.com/richtabor/927437b9a52074acf2eecb90c3108075)).
 
**AC:** 
The "Retrieve your key" and "Need help?" links should be displayed alongside a "new window arrow" icon for each, which is achieved by using the `ExternalLink` core component.

**Current screenshot:** 
<img width="328" alt="Screen Shot 2019-04-28 at 3 41 31 PM" src="https://user-images.githubusercontent.com/1813435/57734966-27a4bb00-7671-11e9-9db9-30699c726ebc.png">

**Comment by richtabor** [comment]
This should be performed from the `issue-495` branch. 👌

**Comment by jrtashjian** [comment]
ExternalLink component added!

![Screen Shot 2019-05-28 at 5 32 06 PM](https://user-images.githubusercontent.com/375788/58513974-b7b52b00-816e-11e9-84bd-06b684baa627.png)

**Comment by richtabor** [comment]
Looks good!
